interface/xtreemfs/OSD.proto

```diff
@@ -354,8 +354,19 @@ message xtreemfs_xloc_set_invalidateRequest {
 }
 
 message xtreemfs_xloc_set_invalidateResponse {
-  required bool is_primary = 1;
-  optional ReplicaStatus status = 2;
+  enum LeaseState {
```

Comment:
I am note sure if i should move this to GlobalTypes, as it is used at the OSD and the MRC? And how to properly name it then. The according enum used in `ReplicaFileState.java` is called `ReplicaState` which is already used in the protobuf interface.